### PR TITLE
ProfileListComponent

### PR DESCRIPTION
**Pull request by Ike1990**

### CODE REVIEW

**Comment by caseymockbee97** [comment]
Looks good to me

File: src/components/ProjectListComponent.jsx [hunk 1]
```diff
@@ -1,5 +1,14 @@
 import React from "react";
+import {useStore} from "../store/store";
 // Things to display: A list of project titles from global store.
+
 export default function ProjectListComponent() {
-  return <div>profile list</div>;
```
Comment (SebastianCaudill) [inline src/components/ProjectListComponent.jsx:4]: is this line still needed because you have 2 returns in this function

Comment (caseymockbee97) [reply src/components/ProjectListComponent.jsx:4]: The stuff in red with a minus is the stuff we took out.

Comment (SebastianCaudill) [reply src/components/ProjectListComponent.jsx:4]: ok

**Pull request merged**